lightning/restore/restore.go

```diff
@@ -1230,17 +1270,20 @@ func (t *TableRestore) postProcess(ctx context.Context, rc *RestoreController, c
 			}
 			rc.saveStatusCheckpoint(t.tableName, WholeTableEngineID, err, CheckpointStatusChecksummed)
 			if err != nil {
-				return errors.Trace(err)
+				return false, errors.Trace(err)
 			}
 		}
+		cp.Status = CheckpointStatusChecksummed
 	}
 
 	// 5. do table analyze
+	doAnalyze := true
```

Comment:
I'm a little confused about `doAnalyze` means this function do or caller should do, hope another name